### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by jmalarcon** [jmalarcon/Win11ClockToggler#19] [Announcements]
Version 3.0 🎉

New year, new version, with:

- **Modernized user interface**
- Support for **Light and Dark Windows themes**
- **New auto-show/hide feature**: just move the cursor over the area where the date/time used to be before being hidden

Thanks to @madninjaskillz for his contributions to this version!

<hr /><em>This discussion was created from the release <a href='https://github.com/jmalarcon/Win11ClockToggler/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by Ander3232** [comment]
Thanks for going to the trouble to create this... Unfortunately, it's not working in Win 11 22H2. The hide options appear disabled in the GUI, and I get your error message, "Maybe you're using a newer version of Windows..." Good luck!

**Reply by jmalarcon** [reply]
Hi. Yes, you're right. I've installed it yesterday and they've changed the way the toolbar UI is built, so the current program it's not able to find the Date/time area.

I've been investigating the new structure and there are two problems:

1. The window/control that contains the Date/Time is located at a different level from where it used to be. This is easily solved and I've managed to locate it in the new location (for 22H2) and in the old location too (for people that has not updated their system). So no problem.
2. This is way worse. When you hide this control, what happens now is that the full toolbar is hidden, not just the Date/Time. I don't know why 🤔 So I don't know if it's possible to restore the functionality because when I hide this everything hides...

I'll keep investigating a little bit further and see if I can get to a solution. If not, I'm afraid that this is the end of the tool for now, unfortunately 😢

I'll keep you posted.

**Comment by Ander3232** [comment]
Hola José – Thanks for your reply!

JMA> Yes, you're right. I've installed it yesterday and they've changed the way the toolbar UI is built, so the current program it's not able to find the Date/time area...

Yes, apparently MS has been up to their usual mischief—changing and moving stuff just for the sake of, you know, changing and moving stuff. 🤷‍♂️

I did a bit more research and found what's currently the next-best thing, as shown in the screenshot I've attached. (In true inexplicable MS fashion, you can edit the "Short time" format in the Region window, but you must click Additional Settings to edit "Short date".) 

By entering these values, all but the two small numerical date digits are hidden. (See screenshot, attached, where I'm running the excellent customizable TClock-Win10. )

Cheers, Andy
 - -
Andy Fielding
Technical Writer, Editor
Vancouver, BC, Canada
____________________________________________________________________________________
 

    On Saturday, March 11, 2023 at 07:52:44 a.m. PST, José M. Alarcón ***@***.***> wrote:  
 
 


Hi. Yes, you're right. I've installed it yesterday and they've changed the way the toolbar UI is built, so the current program it's not able to find the Date/time area.

I've been investigating the new structure and there are two problems:
   
   - The window/control that contains the Date/Time is located at a different level from where it used to be. This is easily solved and I've managed to locate it in the new location (for 22H2) and in the old location too (for people that has not updated their system). So no problem.
   - This is way worse. When you hide this control, what happens now is that the full toolbar is hidden, not just the Date/Time. I don't know why 🤔 So I don't know if it's possible to restore the functionality because when I hide this everything hides...

I'll keep investigating a little bit further and see if I can get to a solution. If not, I'm afraid that this is the end of the tool for now, unfortunately 😢
 
I'll keep you posted.

**Reply by jmalarcon** [reply]
Hi Andy:

Can you paste that screenshot? You've posted this through email and the attachments are not received in GitHub... Thanks.
